### PR TITLE
Update 2016-02-02-create-range-0...n-easily-using-one-line.md

### PR DESCRIPTION
**Pull request by yjcxy12**
## TL;DR;

use `{ length: N }`
## Username

[yjcxy12](https://github.com/yjcxy12)
## Extra

Had a quick test on Chrome dev tool. Solution 4 is much quicker(12 ms on 100,000 elements) than Solution 2 and 3 (both 27ms)

### CODE REVIEW

**Comment by sjfkai** [comment]
What we want was an array like `[0, 1, ..., N-1]`. But `Array.from({ length: N })` just create a empty array with length `N`.

**Comment by yjcxy12** [comment]
@sjfkai That is exactly the same with Solution 2 and 3. 

You do something like 

``` javascript
Array.from({ length: N }, (_, i) => i);
```

**Comment by cjkumaresh** [comment]
@yjcxy12 I feel `Array.from({ length: N }, (_, i) => i);` should be updated in the file instead of just this `Array.from({ length: N })`

**Comment by yjcxy12** [comment]
If we read the whole document, Solution 3 is just a variation of Solution 2, thus in the `One More Thing` session, the creating range function is not included for Solution 3. I see Solution 4 as very similar to Solution 3 in that sense: just a variation of Solution 2, but quicker.

I am willing to add `Array.from({ length: N }, (_, index) => index + 1);`, but should I add it for Solution 3 as well?

**Comment by sjfkai** [comment]
The result of `Solution 3` was difference from `Solution 4`.

``` js
// Solution 3
Array.from(Array(10).keys()); // [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]

// Solution 4
Array.from({ length: 10 }) // [undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined]

```

**Comment by yjcxy12** [comment]
Oh I missed that. Will change.

File: _posts/en/2016-02-02-create-range-0...n-easily-using-one-line.md [hunk 1]
```diff
@@ -41,6 +41,12 @@ Array.from(new Array(N),(val,index)=>index);
 Array.from(Array(N).keys());
 ```
 
+### Solution 4 (requires ES6)
+
+```js
+Array.from({ length: N },(val,index)=>index));
```
Comment (loverajoel) [inline _posts/en/2016-02-02-create-range-0...n-easily-using-one-line.md]: you have an extra `)`

``` js
Array.from({ length: N }, (val, index) => index);
```

**Comment by zenopopovici** [comment]
Stale. Closing.

**Pull request closed without merging**